students/surowiec_marek/lesson_01_basics/apple_sharing.py

```diff
@@ -0,0 +1,5 @@
+# Read the numbers n students, k apples:
```

Comment:
Jeżeli nazwałbyś zmienne bardziej opisowo, np. `students_count` i `apples_count` to nie musiałbyś pisać komentarza i domyślać się w linijce 4 i 5 jaki jest cel obliczeń.
Zalecam poprawkę